kafka_utils/util/monitoring.py

```diff
@@ -92,6 +93,42 @@ def get_consumer_offsets_metadata(
     return result
 
 
+def get_watermark_for_topics_or_regexes(
+    kafka_client,
+    topic
+):
+    """This method:
+        * refreshes metadata for the kafka client
+        * fetches watermarks
+
+    :param kafka_client: KafkaToolClient instance
+    :param topic: the topic
+    :returns: dict <topic>: [ConsumerPartitionOffsets]
+    :raises:
+      :py:class:`kafka_utils.util.error.InvalidOffsetStorageError: upon unknown
+      offset_storage choice.
+    """
+    # Refresh client metadata. We do now use the topic list, because we
```

Comment:
Text copied from offset_get.py, fixing it there and here now. Thanks for pointing this out